components/features/auth/ResetPassword.tsx

```diff
@@ -0,0 +1,140 @@
+import { useToast } from "@/hooks/use-toast";
+import z from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { emailValidation } from "@/lib/validations";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import axiosInstance from "@/lib/axiosInstance";
```

Comment:
Same thing applies here.
```suggestion
import aixos from "@/lib/axiosInstance";
```